layouts/partials/carousel/carousel.css

```diff
@@ -166,6 +171,10 @@
     "background-color:{{ ($.Scratch.Get "settings").item_play_button_hover_color }};" +
 "}" +
 
+".tvp-video:hover > .tvp-video-title {" +
+   "color:{{ ($.Scratch.Get "settings").item_title_font_color_hover }} !important;" +
```

Comment:
Is `!important` really required?

Reply:
yes since it was requested changing `color` property from runTime and we add this inline, without this hover wouldn't work

Reply:
Thanks!